packages/core/src/server/service.test.ts

```diff
@@ -27,7 +27,7 @@ test("port", async (context) => {
     readonlyStore: {} as ReadonlyStore,
   });
 
-  expect(server2.port).toBe(server1.port + 1);
+  expect(server2.port).toBeGreaterThanOrEqual(server1.port + 1);
```

Comment:
This fixes a flake I've seen in CI where a server starts up in a different thread and takes the port directly in between `server1` and `server2` here.